drivers/axi_core/spi_engine/spi_engine.c

```diff
@@ -666,7 +666,8 @@ int32_t spi_engine_init(struct no_os_spi_desc **desc,
 
 	/* Get current data width */
 	spi_engine_read(eng_desc, SPI_ENGINE_REG_DATA_WIDTH, &data_width);
-	eng_desc->max_data_width = data_width;
+	/* Only lower 16 bits are relevant for the actual data-width */
+	eng_desc->max_data_width = data_width & 0xFFFF;
```

Comment:
not mandatory, but i guess u can use:
https://github.com/analogdevicesinc/no-OS/blob/7add1a7756f9f08c62cda688102408d5f8753b41/include/no_os_util.h#L86
and
https://github.com/analogdevicesinc/no-OS/blob/7add1a7756f9f08c62cda688102408d5f8753b41/include/no_os_util.h#L144

Reply:
oh; i thought about using something like that, but i didn't find these on a quick search;
i was probably expecting something closer to the kernel naming